docs/proposals/device-crd.md

```diff
@@ -232,27 +275,23 @@ A `device` instance represents an actual device object. It is like an instantiat
 ```go
 // DeviceSpec represents a single device instance. It is an instantation of a device model.
 type DeviceSpec struct {
```

Comment:
where can we store device instance specific details like device id , mac address etc ?